src/components/OTP.vue

```diff
@@ -309,6 +309,14 @@ export default {
     /** The user type is set as soon as component is created */
     this.userType = this.groupData.userType;
   },
+  mounted() {
+    /** If user already logged in, get from store and redirect to destination */
+    let phoneNumber = this.$store.getters.getUserPhoneNumber;
+    if (phoneNumber) {
+      this.phoneNumberList["0"]["userID"] = phoneNumber;
```

Comment:
any reason we are storing it as `this.phoneNumberList["0"]` instead of `this.phoneNumberList[0]`

Reply:
No reason, I started using it that way and now it persists.

Reply:
It means the same thing right? 
We should use [0] instead of ["0"]!